parley/src/layout/data.rs

```diff
@@ -249,6 +311,7 @@ impl<B: Brush> Default for LayoutData<B> {
             line_items: Vec::new(),
             is_aligned_justified: false,
             alignment_width: 0.0,
+            scratch_clusters: Vec::new(),
```

Comment:
Can this temporary buffer be moved to `LayoutContext` so we don't retain the allocation with the final layout?